custom_model_runner/datarobot_drum/drum/enum.py

```diff
@@ -392,6 +392,20 @@ class TargetType(Enum):
     GEO_POINT = "geopoint"
     VECTOR_DATABASE = "vectordatabase"
 
+    @classmethod
+    def all(cls):
+        return [
+            cls.BINARY,
+            cls.REGRESSION,
+            cls.ANOMALY,
+            cls.UNSTRUCTURED,
+            cls.MULTICLASS,
+            cls.TRANSFORM,
+            cls.TEXT_GENERATION,
+            cls.GEO_POINT,
+            cls.VECTOR_DATABASE,
+        ]
+
```

Comment:
This is redundant, please remove. `Enum` classes can enumerate all types already. i.e. `set(TargetType)`